internal/website/content/howto/blob/_index.md

```diff
@@ -35,12 +35,12 @@ implementations only. See [Concepts: URLs][] for more details.
 ```go
 import _ "gocloud.dev/blob/<driver>"
 
+var bucket *blob.Bucket;
```

Comment:
No semicolon in Go :-).

You'll also want to declare

```
var err error
bucket, err = ... // instead of :=
```

Reply:
Is this a style violation? I tested all code using Go playground and it compiles. I only chose to declare the bucket explicitly to demonstrate the portable type. We don't really need that for error.

Reply:
It doesn't look right to me to do:

```
var bucket *blob.Bucket
bucket, err := ...
```

because the bucket declared on line 1 is shadowed by the new variable (implicitly declared due to `:=`) on line 2. For example, https://play.golang.org/p/6OusvTRTD98. Although, you are right that it will give a compile error if the two variables have different types.

I'd either leave it as it was (e.g., a comment explaining that `bucket` has type `*blob.Bucket`), or declare both `bucket` and `error` and use `=`.

Reply:
If you add statements to your program to print the address of the variable, you will see that it is the same after :=, so there is no shadowing. In fact, https://golang.org/doc/effective_go.html#redeclaration says that this is legal usage, common with err. Does you still mind?

Reply:
You are right; it only shadows if you do it in an a different block (i.e., as part of an `if`).

It's up to you; overall, I find the code odd-looking and nobody would ever write it that way, which probably isn't great for a how-to. Adding a line for `var err error` is also unusual looking, so I think I still vote for either nothing, or a comment like `// bucket is a *blob.Bucket; see below for how to use it.`

Reply:
Done